lib/backend/k8s/syncer.go

```diff
@@ -274,7 +275,7 @@ func (syn *kubeSyncer) readFromKubernetesAPI() {
 			log.Debugf("Incoming NetworkPolicy watch event. Type=%s", event.Type)
 			if needsResync = syn.eventTriggersResync(event); needsResync {
 				// We need to resync.  Break out into the sync loop.
-				log.Warn("Event triggered resync: %+v", event)
+				log.Warnf("Event triggered resync: %+v", event)
```

Comment:
I'm not too fussed about this one.

Looking at the logging code, the String formatting doesn't actually get done unless the correct log level is set, so it won't even have a perf impact.  Plus, this is a low-frequency warning log.